src/SprykerEco/Zed/AmazonQuicksight/Business/ApiClient/AmazonQuicksightApiClient.php

```diff
@@ -0,0 +1,599 @@
+use Generated\Shared\Transfer\EnableQuicksightAnalyticsRequestTransfer;
+use Generated\Shared\Transfer\ErrorTransfer;
+use Generated\Shared\Transfer\QuicksightConsoleTransfer;
+use Generated\Shared\Transfer\QuicksightDeleteUserRequestTransfer;
+use Generated\Shared\Transfer\QuicksightDeleteUserResponseTransfer;
+use Generated\Shared\Transfer\QuicksightDescribeAssetBundleImportJobRequestTransfer;
+use Generated\Shared\Transfer\QuicksightDescribeAssetBundleImportJobResponseTransfer;
+use Generated\Shared\Transfer\QuicksightExperienceConfigurationTransfer;
+use Generated\Shared\Transfer\QuicksightGenerateEmbedUrlRequestTransfer;
+use Generated\Shared\Transfer\QuicksightGenerateEmbedUrlResponseTransfer;
+use Generated\Shared\Transfer\QuicksightListUsersRequestTransfer;
+use Generated\Shared\Transfer\QuicksightListUsersResponseTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceCredentialPairTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceCredentialsTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceMariaDbParametersTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceParametersTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersDataSourceVpcConnectionPropertiesTransfer;
+use Generated\Shared\Transfer\QuicksightOverrideParametersTransfer;
+use Generated\Shared\Transfer\QuicksightOverridePermissionsAnalysisTransfer;
+use Generated\Shared\Transfer\QuicksightOverridePermissionsDashboardTransfer;
+use Generated\Shared\Transfer\QuicksightOverridePermissionsDataSetTransfer;
+use Generated\Shared\Transfer\QuicksightOverridePermissionsDataSourceTransfer;
+use Generated\Shared\Transfer\QuicksightOverridePermissionsTransfer;
+use Generated\Shared\Transfer\QuicksightPermissionsTransfer;
+use Generated\Shared\Transfer\QuicksightStartAssetBundleImportJobRequestTransfer;
+use Generated\Shared\Transfer\QuicksightStartAssetBundleImportJobResponseTransfer;
+use Generated\Shared\Transfer\QuicksightUpdateUserRequestTransfer;
+use Generated\Shared\Transfer\QuicksightUpdateUserResponseTransfer;
+use Generated\Shared\Transfer\QuicksightUserRegisterRequestTransfer;
+use Generated\Shared\Transfer\QuicksightUserRegisterResponseTransfer;
+use Generated\Shared\Transfer\QuicksightUserTransfer;
+use Generated\Shared\Transfer\ResetQuicksightAnalyticsRequestTransfer;
+use Generated\Shared\Transfer\UserTransfer;
+use SprykerEco\Zed\AmazonQuicksight\AmazonQuicksightConfig;
+use SprykerEco\Zed\AmazonQuicksight\Business\Formatter\AmazonQuicksightRequestDataFormatterInterface;
+use SprykerEco\Zed\AmazonQuicksight\Business\Mapper\AmazonQuicksightMapperInterface;
+use SprykerEco\Zed\AmazonQuicksight\Dependency\External\AmazonQuicksightToAwsQuicksightClientInterface;
+
+class AmazonQuicksightApiClient implements AmazonQuicksightApiClientInterface
```

Comment:
* The class in unmaintainable, I would suggest at least to split it by the domain entities. (User/AssetBundle).

Reply:
Resolved.